app/views/layouts/_labs_main.erb

```diff
@@ -122,7 +122,7 @@
                     <nav>
                         <ul class="pull-right list-inline">
                             <li><a href="/terms-of-use">Terms of use</a></li>
-                            <li><a href="/privacy-policy">Privacy policy</a></li>
+                            <li><a href="/privacy_policy">Privacy policy</a></li>
                             <li><a href="/cookie-policy">Cookie policy</a></li>
```

Comment:
This is still a 404, but can use http://theodi.org/cookie-policy as a guide. I don't think we store anything beyond the usual Google Analytics stuff - but might be worth checking